exopy/app/log/widgets.enaml

```diff
@@ -0,0 +1,28 @@
+# -*- coding: utf-8 -*-
+# -----------------------------------------------------------------------------
+# Copyright 2015-2018 by Exopy Authors, see AUTHORS for more details.
+#
+# Distributed under the terms of the BSD license.
+#
+# The full license is in the file LICENCE, distributed with this software.
+# -----------------------------------------------------------------------------
+"""Widget used to display the rotating log.
+
+"""
+from enaml.widgets.api import (Container, Dialog, Label, ScrollArea)
+
+
+enamldef LogDialog(Dialog):
+    """Dialog presenting the rotating log.
+
+    """
+    title = 'Log viewer'
+
+    attr rotating_log = "Default log"
```

Comment:
`log` as suggested above or `log_content` Also could you add a minimalist comment (starting with a `#:`) above.